lib/rototiller/utilities/collector.rb

```diff
@@ -0,0 +1,65 @@
+require 'rototiller/utilities/flag'
+require 'rototiller/utilities/env_var'
+require 'forwardable'
+
+class ParamCollection
+
+  # This may be useful if we use individual ParamCollection objects for EnvVars and Flags
+  extend Forwardable
```

Comment:
this is probably better for things like Forwardable rather than <, which i've used in the past since this isn't _a_ forwardable.  :neutral_face: